cloud-sql/mysql/activerecord/bin/run_tests

```diff
@@ -0,0 +1,9 @@
+#! /bin/sh
+
+# Run the tests with default values for all
+# environment variable
```

Comment:
Kokoro still thinks the tests are passing, even though there are connection errors in the logs. Maybe try adding `set -e` to these scripts to fix it?
![image](https://user-images.githubusercontent.com/19631367/104672395-94692780-5694-11eb-9a4c-8a224557221d.png)